rclcpp/include/rclcpp/node.hpp

```diff
@@ -117,6 +99,14 @@ class Node : public std::enable_shared_from_this<Node>
   rclcpp::callback_group::CallbackGroup::SharedPtr
   create_callback_group(rclcpp::callback_group::CallbackGroupType group_type);
 
+  using CallbackGroup = rclcpp::callback_group::CallbackGroup;
```

Comment:
I just removed it and the following line, not sure why I added that there. Might have been a copy-paste issue, thanks for pointing it out. 28a2cfc